dome9/data_source_dome9_cloudaccount_kubernetes.go

```diff
@@ -44,6 +44,45 @@ func dataSourceCloudAccountKubernetes() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"runtime_protection": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": {
+							Type:     schema.TypeBool,
```

Comment:
Clarify model plan please.

Reply:
Our intention here was to structure the features' configurations as complex objects from the beginning, so that future expansions are easy (e.g. if we add some additional configuration parameter we just expand the relevant object).

If you see downsides to this approach, we can discuss it further.